ledgereth/transactions.py

```diff
@@ -87,9 +124,11 @@ def create_transaction(
     to: bytes,
     value: int,
     gas: int,
-    gas_price: int,
     nonce: int,
     data: Text,
+    gas_price: int = None,
```

Comment:
technically this is `Optional[int]` ?

Reply:
Should've probably defaulted to `0` on this one now too.  Thanks